README.md

```diff
@@ -62,6 +62,17 @@ spec:
     provider: <GCP_OR_AWS>
     region: <AWS_REGION>
 ```
+
+For local snapshot of CStor volume through velero, Refere `example/06-local-volumesnapshotlocation.yaml` to configure `VolumeSnapshotLocation`.
```

Comment:
to be consistent with 'cloud backup', should we call this as 'local backup'? before changing, lets get opinion from @kmova as well

Reply:
That may create confusion with local copy of the snapshot.

Reply:
For creating local backup using cStor Volume snapshot feature, refer `example/06-local-volumesnapshotlocation.yaml` to configure `VolumeSnapshotLocation`.